skimage/data/__init__.py

```diff
@@ -178,27 +178,31 @@ def logo():
 
 
 def microaneurysms():
-        """Gray-level "microaneurysms" image.
-
-        Detail from an image of the retina (green channel).
-        The image is a crop of image 07_dr.JPG from the
-        High-Resolution Fundus (HRF) Image Database:
-        https://www5.cs.fau.de/research/data/fundus-images/
-
-        Returns
-        -------
-        microaneurysms : (102, 102) uint8 ndarray
-            Retina image with lesions.
-
-        References
-        ----------
-        .. [1] Budai, A., Bock, R, Maier, A., Hornegger, J.,
-               Michelson, G. (2013).  Robust Vessel Segmentation in Fundus
-               Images. International Journal of Biomedical Imaging, vol. 2013,
-               2013.
-               :DOI:`10.1155/2013/154860`
-        """
-        return load("microaneurysms.png")
+    """Gray-level "microaneurysms" image.
+
+    Detail from an image of the retina (green channel).
+    The image is a crop of image 07_dr.JPG from the
+    High-Resolution Fundus (HRF) Image Database:
+    https://www5.cs.fau.de/research/data/fundus-images/
+
+    Notes
```

Comment:
I feel like Notes should come after Returns, but such a nit. so who cares.

Reply:
Nevermind, seems notes come before returns.

Reply:
https://github.com/scikit-image/scikit-image/pull/3625/files

Reply:
@hmaarrfk what made you say that it comes before?

https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

Reply:
Ah, why didn't I follow that link? 🤦‍♂️ Anyway, possibly we should change this, but I agree that it should be a separate PR.